src/core/constants.js

```diff
@@ -153,14 +153,10 @@ export const SET_VIEW_CONTEXT = 'SET_VIEW_CONTEXT';
 export const CLOSE_INFO = 'CLOSE_INFO';
 export const SHOW_INFO = 'SHOW_INFO';
 
-// Theme preview actions.
+// Theme actions.
 export const THEME_INSTALL = 'InstallBrowserTheme';
-export const THEME_PREVIEW = 'PreviewBrowserTheme';
-export const THEME_RESET_PREVIEW = 'ResetBrowserThemePreview';
 export const validThemeActions = [
   THEME_INSTALL,
```

Comment:
It seems a bit odd to have an array with only a single item in it for `validThemeActions`. If there's only one possible valid action, why do we need this at all? That's a rhetorical question, as I can see from the code in `themeInstall.js` why we need this, but maybe that code can change to remove the need for this entirely. `themeInstall.js` includes a function called `themeAction` which accepts an `action` as an argument, but if there is only one valid `action` now, perhaps that function could be renamed to `installTheme` because that's the only thing it's ever going to do, and it would no longer require an `action` argument. What do you think about that, @rebmullin?